src/main/java/cloud/fogbow/rcs/constants/ConfigurationPropertyDefaults.java

```diff
@@ -6,4 +6,8 @@
     // RCS CONF DEFAULTS
     public static final String BUILD_NUMBER = "[testing mode]";
     public static final String CACHE_EXPIRATION_TIME_DEFAULT = Long.toString(TimeUnit.HOURS.toMinutes(24)); // One day
+    
+    // INTERCOMPONENT CONF DEFAULT
+    public static final String XMPP_CSC_PORT = Integer.toString(5347);
+    public static final String XMPP_TIMEOUT = Long.toString(TimeUnit.SECONDS.toMillis(5)); // the reference value is 5 seconds
```

Comment:
this was throwing a NumberFormatException in the integration branch when no cache_expiration is set on the config file. can you please check if this is fixed in this pr?

Reply:
This problem is a bug that is related to the implementation of the cache while getting your properties. Should I make the correction available in this PR?

Reply:
yes please :)

Reply:
Fixed... The change is found in the **MemoryBasedCach** class constructor.